hotel-management/src/components/Dropdown/DropdownFilterMenu.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+
+import { DropdownItem } from '@/types/dropdown';
+
+import { Button } from '../Button';
+import './index.css';
+
+type DropdownProps = {
+  label: string;
+  buttonIcon?: JSX.Element;
+  items?: DropdownItem[];
```

Comment:
`buttonIcon` should be optional because the button may not need an icon, but `items` I now think it should be required as the dropdown shouldn't be empty